test/helpers/mocks.dart

```diff
@@ -49,3 +53,15 @@ class MockTransactionNotifier extends AutoDisposeFamilyAsyncNotifier<
     TransactionProviderParameters> with Mock implements TransactionNotifier {
   MockTransactionNotifier();
 }
+
+class MockAccountBalanceNotifier
+    extends AutoDisposeFamilyAsyncNotifier<AccountBalance?, List<Pfi>>
+    with Mock
+    implements AccountBalanceNotifier {
+  final AccountBalance? accountBalance;
+
+  MockAccountBalanceNotifier(this.accountBalance);
+
+  @override
+  FutureOr<AccountBalance?> build(List<Pfi> arg) async => accountBalance;
+}
```

Comment:
@wesbillman what's the correct way to override this AsyncNotifierProvider with a specific `AccountBalance`?